study-builder/src/main/java/org/broadinstitute/ddp/studybuilder/StudyBuilderCli.java

```diff
@@ -150,6 +152,13 @@ private void run(String[] args) throws Exception {
             execute(builder::runEvents, isDryRun);
             log("done");
             return;
+        } else if (cmd.hasOption("only-labeled-events")) {
+            log("executing events setup...");
```

Comment:
Maybe we can be a bit more specific?
```suggestion
            log("executing labeled events setup...");
```